packages/server/src/api/controllers/application.ts

```diff
@@ -450,37 +451,51 @@ export const revertClient = async (ctx: any) => {
   ctx.body = app
 }
 
-const destroyApp = async (ctx: any) => {
```

Comment:
For peace of mind around dev vs prod app ids, it'd be nice to see tests like:

```ts
describe("DELETE /api/applications/:id", () => {
  it("deletes prod and dev apps using dev app id", () => {})
  it("deletes prod and dev apps using prod app id", () => {})
})

describe("POST /api/applications/:id/publish", () => {
  it("publishes with dev app id", () => {})
  it("publishes with prod app id", () => {})
})

describe("POST /api/applications/:id/unpublish", () => {
  it("unpublishes with dev app id", () => {})
  it("unpublishes with prod app id", () => {})
  it("rejects when app is not published", () => {})
})
```

Reply:
Yep, they are in progress - with each of the different scenarios. Just been pushing changes incrementally to make sure I'm not breaking the existing endpoints

Reply:
👍 👍